app/models/metric/ci_mixin/capture.rb

```diff
@@ -7,6 +7,7 @@ def perf_capture_object(targets = nil)
     end
   end
 
+  # delegate currently only used on Ems
```

Comment:
How do VMs/Hosts call in to the MetricsCapture class then?  At least in specs I see a lot of `vm.perf_collect_metrics('realtime')` and `host.perf_collect_metrics("realtime")`

Reply:
I see a number of these in specs.

In the code, we explicitly reference `performance_capture_object.perf_collection_metrics`

This is pulled out so we can put multiple targets in the `perf_capture_object` and capture lots of targets at the same time.